incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,12 @@
+//////////  S V G  M I X I N  //////////////////////////////
```

Comment:
I think there should be 3 spaces between the words. 2 spaces looks like 1 space so the 2 words seem like 1 but with odd character spacing.

Yes, I commented on your comment style =)